config/services.php

```diff
@@ -40,12 +40,13 @@
     ],
 
     'discord' => [
-        'invite_url' => env('DISCORD_INVITE_URL'),
-        'guild_id' => env('DISCORD_GUILD_ID'),
-        'token' => env('DISCORD_TOKEN'),
-        'client_id' => env('DISCORD_CLIENT'),
-        'client_secret' => env('DISCORD_SECRET'),
-        'redirect_uri' => env('DISCORD_REDIRECT_URI'),
+        'invite_url' => env('DISCORD_INVITE_URL', null),
+        'guild_id' => env('DISCORD_GUILD_ID', null),
+        'token' => env('DISCORD_TOKEN', null),
+        'client_id' => env('DISCORD_CLIENT', null),
+        'client_secret' => env('DISCORD_SECRET', null),
+        'redirect_uri' => env('DISCORD_REDIRECT_URI', null),
+        'base_discord_uri' => env('DISCORD_API_BASE', 'https://discordapp.com/api/v6')
```

Comment:
I had this URI hard coded as it was a way of showing that the Library for Discord had been written for V6 of the API. Might be worth just commenting that into the library if we are extracting the URI to a config item